src/Components/Icon.tsx

```diff
@@ -26,11 +26,13 @@ export interface IconProps
   onClick?: () => void
 }
 
-const Icon: React.SFC<IconProps> = props => (
-  <div className={props.className} style={props.style} onClick={props.onClick}>
-    {icons[props.name]}
-  </div>
-)
+const Icon: React.SFC<IconProps> = ({
+  color,
+  font,
+  fontSize,
```

Comment:
I think these props aren't needed here. So, it's a bit of an odd case. `color`, `font`, and `fontSize` are actually being handled by the `styled` wrapper that's put around this component (see [#L37](https://github.com/artsy/reaction/pull/786/files#diff-8299e56061299f400f96969da00c2b56L35)). It shouldn't actually receive these props.